WordPress/Classes/ViewRelated/Gutenberg/Processors/GutenbergCoverUploadProcessor.swift

```diff
@@ -0,0 +1,63 @@
+import Foundation
+import Aztec
+
+class GutenbergCoverUploadProcessor: Processor {
+    public typealias InnerBlockProcessor = (String) -> String?
+
+    let mediaUploadID: Int32
+    let remoteURLString: String
+    let serverMediaID: Int
+
+    init(mediaUploadID: Int32, serverMediaID: Int, remoteURLString: String) {
+        self.mediaUploadID = mediaUploadID
+        self.serverMediaID = serverMediaID
+        self.remoteURLString = remoteURLString
+    }
+
+    lazy var coverBlockProcessor = GutenbergBlockProcessor(for: "wp:cover", replacer: { coverBlock in
```

Comment:
It will be good to have strings like `wp:cover` and `background-image:url(file:///` be defined as constants, like we defined in the `GutenbergGalleryUploadProcessor`